### PR TITLE
fix: plugin wrapper flash

### PR DESCRIPTION
**Pull request by yanzhihong23**
## Description

Closes [MF-171](https://mask.atlassian.net/browse/MF-171)

## Type of change

- [ ] Documentation
- [ ] Code refactoring (Restructuring existing code w/o changing its observable behavior)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (a fix or feature that would make something no longer possible to do/require old user must upgrade their Mask Network to this new version)

## Checklist

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
  - [ ] I have removed all in development `console.log`s
  - [ ] I have removed all commented code.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have read [Internationalization Guide](https://github.com/DimensionDev/Maskbook/blob/develop/docs/i18n-guide.md) and moved text fields to the i18n JSON file.

If this PR depends on external APIs:

- [ ] I have configured those APIs with CORS headers to let extension requests get passed. 
  - chrome extension: `chrome-extension://[id]`
  - firefox extension: `moz-extension://[id]`
- [ ] I have delegated all web requests to the background service via the internal RPC bridge.

### CODE REVIEW

**Comment by wenluomask** [comment]
Checked.

**Review by Jack-Works — APPROVED** [review]

File: packages/plugin-infra/src/hooks/usePluginWrapper.tsx [hunk 1]
```diff
@@ -24,7 +24,7 @@ export function usePluginWrapper(open: boolean, options?: { width?: number; name
 
     useEffect(() => {
         setWrap(open)
-        return () => setWrap(false)
+        return () => setWrap(open)
```
Comment (Jack-Works) [inline packages/plugin-infra/src/hooks/usePluginWrapper.tsx:27]: hmm wait, this is strange...

Comment (guanbinrui) [reply packages/plugin-infra/src/hooks/usePluginWrapper.tsx:27]: Why does it let the child component decide how the parent component should behave?

Comment (Jack-Works) [reply packages/plugin-infra/src/hooks/usePluginWrapper.tsx:27]: because it behave like a context 👀

Comment (guanbinrui) [reply packages/plugin-infra/src/hooks/usePluginWrapper.tsx:27]: @yanzhihong23 Is this operation useless?

**Comment by guanbinrui** [comment]
@DimensionDev/qa

**Pull request merged**